web/main.js

```diff
@@ -1,164 +1,261 @@
-let metadata, reviews, discussions;
+{
+  let readReviews, readDiscussions;
+  let metadata, reviews, discussions;
 
-const getData = async (file) => {
+  window.markAllAsRead = (event) => {
+    event.stopPropagation();
 
-  var headers = new Headers();
-  headers.append("pragma", "no-cache");
-  headers.append("cache-control", "no-cache");
+    { // Discussions
+      const discussionCtnr = document.querySelector("#discussion-container");
+      const discussionStars = discussionCtnr.querySelectorAll(".unread-star");
+      for (const discussionStar of discussionStars) {
+        const key = discussionStar.getAttribute("data-key");
+        readDiscussions[key] = true;
+        discussionStar.classList.add("hidden");
+      }
 
-  const res = await fetch(`data/${file}`, {
-    method: "GET",
-    headers: headers
-  });
+      setLocalStorageJSONObject("readDiscussions", readDiscussions);
+    }
+
+    { // Reviews
+      const reviewCtnr = document.querySelector("#review-container");
```

Comment:
I prefer writing out abbreviations, even though it makes the code look like more quantity.

Reply:
In this case it's following the existing convention in the file, e.g. from `renderReviews`. Feel free to rename across the file if you please, I'll prolly keep this as is on my end

Reply:
Actually I'm finding myself going through and looking for braces for consistency so I'll rename these as I go.

This exercise makes me think that it'd be good to use a linter for this 😛 but maybe overkill for 1 file

Reply:
Expanded all abbreviations in aadf315411b2162df5332430ecf89e1a5a9f61e9